.github/workflows/erlang.yml

```diff
@@ -10,17 +10,21 @@ jobs:
 
     strategy:
       matrix:
-        otp: [21.3, 22.3, 23.0.4, 23.2.7.0, 24.0]
-    container:
-      image: erlang:${{ matrix.otp }}-alpine
+        otp: ['21.3', '22.3', '23.3']
+        rebar: ['3.15.2']
+        include:
+        - otp: '24.0'
+          rebar: '3.16.1'
```

Comment:
```suggestion
        otp: ['22.3', '23.3', '24.0']
        rebar: ['3.16.1']
        include:
        - otp: '21.3'
          rebar: '3.15.2'
```

Reply:
This way, when the time to stop supporting 21.3 comes you only have to delete stuff 😄